Makefile

```diff
@@ -44,14 +46,19 @@ generate:
 # Build the docker image
 docker-build: resolve test
 	docker build . -t ${IMG}
+	docker tag ${IMG} ${IMG-CI}
 	@echo "updating kustomize image patch file for manager resource"
-	sed -i'' -e 's@image: .*@image: '"${IMG}"'@' ./config/default/manager_image_patch.yaml
+	sed -i'' -e 's@image: .*@image: '"${IMG-CI}"'@' ./config/default/manager_image_patch.yaml
```

Comment:
Why the sed is needed? Can you explain the tooling?

Reply:
sed is just ensuring that the image is up to date with new build image. This is generated by kubebuilder, I do not want to change there anything before we will know how we will handle the final files generation. This PR is only about CI so I do not want to change the default behavior.